packages/techdocs-common/package.json

```diff
@@ -56,6 +56,7 @@
     "mime-types": "^2.1.27",
     "mock-fs": "^4.13.0",
     "p-limit": "^3.1.0",
+    "pkgcloud": "^2.2.0",
```

Comment:
I did not know about this package. https://github.com/pkgcloud/pkgcloud#storage

Turns out we could just use `pkgcloud` as an abstraction over all cloud providers in TechDocs. But not critical as of now.

Reply:
There is no official OpenStack SDK for node.js and we found this package in OpenStack wiki other packages outdated (last commit 4 years ago). This package maintainer not update last commit 17 months ago i think its not suitable for backstage. We plan to create updated OpenStack SDK they use ApplicationId and Secret for Auth.